scripts/ci/pack_soljson.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+SOLJSON_JS="$1"
+SOLJSON_WASM="$2"
+SOLJSON_WASM_SIZE=$(wc -c "${SOLJSON_WASM}" | cut -d ' ' -f 1)
+OUTPUT="$3"
+
+(( $# == 3 )) || { >&2 echo "Usage: $0 soljson.js soljson.wasm packed_soljson.js"; exit 1; }
+
+# If this changes in an emscripten update, it's probably nothing to worry about,
+# but we should double-check when it happens and adjust the tail command below.
+[ "$(head -c 5 "${SOLJSON_JS}")" == "null;" ] || { >&2 echo 'Expected soljson.js to start with "null;"'; exit 1; }
```

Comment:
If we're not aiming for portability (the script uses Bash) I'd use `[[ ]]`. In this case it's not strictly necessary but I 'd stick to it as a general rule.
```suggestion
[[ $(head -c 5 "${SOLJSON_JS}") == "null;" ]] || { >&2 echo 'Expected soljson.js to start with "null;"'; exit 1; }
```